lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,14 @@
-
 def palindrome_permutation?(string)
+  # I am using this index-iteration strategy here and in permutations.rb because I'm operating on the assumption that it would be more expensive, space-wise, to make an array of characters using string.chars. Is this true?
```

Comment:
You do not need an array of characters. Ruby Strings are have `#each_char` that you can use to iterate over the characters.